code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -94,14 +94,14 @@
 
 	var/list/turf/synd_spawn = list()
 
-	for(var/thing in GLOB.landmarks_list)
-		var/obj/effect/landmark/A = thing
-		if(A.name == "Syndicate-Spawn")
-			synd_spawn += get_turf(A)
-			qdel(A)
-			continue
-
-	var/obj/effect/landmark/nuke_spawn = locate("landmark*Nuclear-Bomb")
+	for(var/obj/effect/landmark/spawner/syndie/S in GLOB.landmarks_list)
+		synd_spawn += get_turf(S)
+		qdel(S)
+		continue
+
+	var/obj/effect/landmark/nuke_spawn
+	for(var/obj/effect/landmark/spawner/nuclear_bomb in GLOB.landmarks_list)
+		nuke_spawn = get_turf(nuclear_bomb)
```

Comment:
why not
```suggestion
	var/obj/effect/landmark/nuke_spawn = get_turf(locate(/obj/effect/landmark/spawner/nuclear_bomb))
```
?

Reply:
lol why did I do it in such a roundabout way, thanks